gunpowder/version_info.py

```diff
@@ -1,7 +1,7 @@
 __major__   = 1
 __minor__   = 2
 __patch__   = 2
-__tag__     = ''
+__tag__     = '-raygun-dev'
```

Comment:
That should probably not be part of the PR.

Reply:
Agreed. I had it in during dev, and forgot to remove it. Sorry about that!